mail_optional_follower_notification/wizard/mail_compose_message_view.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="email_compose_message_wizard_form" model="ir.ui.view">
+        <field name="model">mail.compose.message</field>
+        <field name="inherit_id" ref="mail.email_compose_message_wizard_form" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//widget[@name='mail_composer_recipient_list']"
+                position="before"
+            >
+                <field
+                    name="notify_followers"
```

Comment:
This field is missing it's label. Without it, the checkbox doesn't have any text in the mail composer:
<img width="980" alt="Screenshot 2024-10-31 at 09 46 51" src="https://github.com/user-attachments/assets/08e81571-8181-4bb2-ae73-5a2c668b1d12">